### DISCUSSION TITLE
工程链接什么时候能发啊，orz

### DISCUSSION THREAD

**Discussion by Nex-to-me** [peng-zhihui/HelloWord-Keyboard#45] [General]

**Comment by peng-zhihui** [comment]
这周都在出差，估计下周末吧

**Comment by Nex-to-me** [comment]
如果是98键的将是绝杀，可惜我自己不会改，只能复刻了




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年8月7日(星期天) 晚上10:36
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [peng-zhihui/HelloWord-Keyboard] 工程链接什么时候能发啊，orz (Discussion #45)





 
这周都在出差，估计下周末吧

**Comment by peng-zhihui** [comment]
晚点有空我会补充其他配列的PCB，或者更新一个自己修改PCB的教程。

**Reply by djkkqqkk** [reply]
大佬，希望将键盘分成4个区，主键盘区（这里分成2半，一半ASDFG，一半HJKL）、控制键区、数字键区。
可以让不同的人群使用或DIY。
有些朋友喜欢英雄联盟，射击游戏，基本上也只是使用（主要是便携）
ASDFG这一边的键区
